msg-socket/src/req/socket.rs

```diff
@@ -191,19 +187,26 @@ where
             timer
         });
 
+        // Create connection manager
+        let conn_manager = ConnManager::new(
+            transport,
+            endpoint,
+            conn_ctl,
+            Arc::clone(&self.state.transport_stats),
+            self.options.auth_token.clone(),
+            span.clone(),
```

Comment:
right now ConnManager and driver use the same span. I can make two separate ones if it's cleaner?